opentelemetry/src/metrics/meter.rs

```diff
@@ -120,41 +99,80 @@ impl Meter {
         )
     }
 
-    /// Creates a new integral `SumObserverBuilder` instrument with the given name
-    /// and callback
-    pub fn u64_sum_observer<T, F>(&self, name: T, callback: F) -> SumObserverBuilder<'_, u64>
+    /// Creates a new floating point `UpDownSumObserverBuilder` instrument with the
```

Comment:
Emm, I think we should probably change it like below as `UpDownSumObserver` is an instrument but `UpDownSumObserverBuilder` technically isn't.
```suggestion
    /// Creates a `UpDownSumObserverBuilder` for the floating point `UpdownSumObserver` instrument with the
```

Reply:
I did a pass through all the comments for consistency.